src/components/NavBar/index.tsx

```diff
@@ -144,9 +147,10 @@ const styles = (theme: Theme) =>
 
     drawerContent: {
       backgroundColor: theme.palette.primary.main,
-      padding: 16,
+      paddingTop: 16,
```

Comment:
Are you sure that this is not affecting the UI on desktop?

Reply:
Should be safe, because the class is only used in mobile screen (drawer that appears when users click the button to show the menu) and it's hidden in the desktop

Reply:
Maybe we could prepend the classname with `drawer` -> `mobileDrawerContent` so its more clear its mobile? Just a suggestion, since `drawer` is normally associated with mobile.